docs/contributing/development.md

```diff
@@ -0,0 +1,52 @@
+# Development Guidelines
+
+## Getting Started
+
+### Prerequisites
+
+Before you begin development, ensure you have the following prerequisites:
+
+- A working installation of Go (version 1.16 or later).
```

Comment:
I think the current device plugin repo has claimed go 1.23 as the current golang version. what do you think @AMD-melliott @y2kenny-amd ?

Reply:
This is correct.  go.mod points to 1.23.